tests/monitoring/monitoring.go

```diff
@@ -320,6 +327,67 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		})
 	})
 
+	Context("VM status metrics", func() {
+		newVirtualMachine := func() *v1.VirtualMachine {
+			vmi := tests.NewRandomVMI()
+			return tests.NewRandomVirtualMachine(vmi, false)
```

Comment:
Please consider to set "running" to `true`. Then, you won't have to explicitly start the VM in the tests below.

Reply:
changed